README.md

```diff
@@ -7,6 +7,15 @@ switch between database connections. We've made the implementation very small, a
 
 ActiveRecord Shards has been used and tested on Rails 3.2, 4.2 and 5.0 and has in some form or another been used in production on a large Rails app for several years.
 
+- [Installation](#installation)
+- [Configuration](#configuration)
+- [Migrations](#migrations)
+  - [Example](#example)
+    - [Shared Model](#create-a-table-for-the-shared-not-sharded-model)
```

Comment:
maybe we should call this `global` or so to make it less likely to be confused visually `shared` <-> `sharded`

Reply:
Yeah, maybe a followup PR. Stuck with the existing naming to be consistent.